README.md

```diff
@@ -1,12 +1,20 @@
-# Exodus Privacy
+# Exodus - Android Application
 
 [![Build Status](https://github.com/Exodus-Privacy/exodus-android-app/actions/workflows/main.yml/badge.svg?branch=master)](https://github.com/Exodus-Privacy/exodus-android-app/actions/workflows/main.yml) [![Language grade: Java](https://img.shields.io/lgtm/grade/java/g/Exodus-Privacy/exodus-android-app.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Exodus-Privacy/exodus-android-app/context:java)
 
-Exodus Privacy is an Android application.
-Exodus Privacy application let you know what trackers are embedded in apps installed on your smartphone. It let you also know the permissions required by any apps on your smartphone. 
+Exodus is an Android application that let you know what trackers are embedded in apps installed on your smartphone using [the εxodus plateform](https://github.com/Exodus-Privacy/exodus). It let you also know the permissions required by any apps on your smartphone.
+
 It helps you to take your privacy back!
 
 <a href="https://f-droid.org/packages/org.eu.exodus_privacy.exodusprivacy/" target="_blank">
 <img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png" alt="Get it on F-Droid" height="80"/></a>
 <a href="https://play.google.com/store/apps/details?id=org.eu.exodus_privacy.exodusprivacy" target="_blank">
 <img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png" alt="Get it on Google Play" height="80"/></a>
+
+## Getting in touch with us
+
+If you want to help us improve this application, you can:
+
+- Use [the issues](https://github.com/Exodus-Privacy/exodus/issues)
+- Join us on our [IRC channel #exodus-android-app on Libera.chat](https://web.libera.chat/?nick=webguest?#exodus-android-app)
+- Join us on our [Matrix channel #exodus-android-app:matrix.org](https://matrix.to/#/#exodus-android-app:matrix.org)
```

Comment:
Can we somehow make it more clear that this is the same channel, but a different way to join?

Reply:
Better?

Reply:
Yes, thanks!